### PR TITLE
Deliver Oracle for VET/USD + VTHO/USD Pricing

### PR DESCRIPTION
**Pull request by ifavo**
This pull request contains the delivery of all 7 milestones for https://github.com/vechain/grant-program/pull/214

The oracle has been fully developed:

1. Public Repository with all instructions is available on: https://github.com/vechain-energy/oracle.vechain.energy
2. Contracts and feeding is done since the beginning of the year and can be checked on: https://status.oracle.vechain.energy

Thank you for your trust and support on building things!

### CODE REVIEW

File: milestone-delivery/deliveries/oracle.vechain.energy-M1-M2.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+* **Milestone Number:** 1 - 7
+
+## Milestone 1 - 7
+
+
+| Number | Deliverable | Link | Notes |
+|-|-|-|-|
+| 1.1 | Contract | https://github.com/vechain-energy/oracle.vechain.energy/tree/main/contracts |
+| 1.2 | TestNet | https://explore.vechain.org/accounts/0x49ec7192bf804abc289645ca86f1ed01a6c17713
+| 1.3 | MainNet | https://explore-testnet.vechain.org/accounts/0xdccaabd81b38e0deef4c202bc7f1261a4d9192c6
```
Comment (libotony) [inline milestone-delivery/deliveries/oracle.vechain.energy-M1-M2.md:16]: Network mismatch.

Comment (ifavo) [reply milestone-delivery/deliveries/oracle.vechain.energy-M1-M2.md]: thank you 🙏

**Comment by ifavo** [comment]
> As per our rule, each milestone should be in a separate file. You can find more information at https://github.com/vechain/grant-program/tree/master/milestone-delivery#milestone-delivery-process. Could you kindly separate them accordingly?

I separated it into seven separated files, please let me know if anything needs to be changed or is missing. 🙏

**Review by libotony — APPROVED** [review]

**Review by NecoSherry — APPROVED** [review]
Thank you for your delivery. We will proceed with the payment for your application promptly. We appreciate your patience.

**Pull request merged**